todo/src/components/Action.js

```diff
@@ -0,0 +1,17 @@
+export const Add_Todo = 'Add_Todo';
```

Comment:
Generally, the name of a constant like this would be in upper case snake case just like how you name the TOGGLE_TODO constant. So,
```
  export const ADD_TODO = 'ADD_TODO';
```